constant/style/queries.ts

```diff
@@ -0,0 +1,6 @@
+import { laptop, mobile, tablet } from "./breakpoints";
+
+export const mobileAndUnder = `(max-width: ${mobile}px)`;
+export const tabletAndUnder = `(max-width: ${tablet}px)`;
+export const laptopAndUnder = `(max-width: ${laptop}px)`;
```

Comment:
This is an interesting one. I used to do it like this in the past, and I ended up deciding to do it with separate constants instead for a couple reasons. Firstly, I'm literally just too lazy to type `queries.` in general, and doing it like this lets you avoid that. More importantly though is the difference in editor autocomplete that you get with this method. With these named exports for each query, you can write `@media ${ta` and vs code will autocomplete that to `@media ${tabletAndUnder}` and auto import the constant for you. If you keep them in objects, vs code can't predict that what you are looking for is `queries.tabletAndUnder` and it can't do that super convenient autocomplete.